<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Baker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="titleInfoBox">
      <property name="title">
       <string>Title info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="tPathLabel">
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vPathLabel">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="tTitleLabel">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vTitleLabel">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="tNumLabel">
           <property name="text">
            <string>Number of episodes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vNumLabel">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="tBitLabel">
           <property name="text">
            <string>Pixel color depth:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vBitLabel">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="tAudioLabel">
           <property name="text">
            <string>Audio tracks availiable:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vAudioLabel">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tSubLabel">
           <property name="text">
            <string>Subtitles availiable:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vSubLabel">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="settingsTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="videoLayout">
          <item>
           <widget class="QCheckBox" name="convertVideoCheck">
            <property name="text">
             <string>Convert video</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="eightBitButton">
            <property name="text">
             <string>to 8-bit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="tenBitButton">
            <property name="text">
             <string>to10-bit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="audioLayout">
          <item>
           <widget class="QCheckBox" name="audioCheck">
            <property name="text">
             <string>Add audio track</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="audioBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="audioPreviewButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="subtitlesLayout">
          <item>
           <widget class="QCheckBox" name="subCheck">
            <property name="text">
             <string>Add subtitles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="subBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="rangeFromLabel">
            <property name="text">
             <string>Process episodes from</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="startBox"/>
          </item>
          <item>
           <widget class="QLabel" name="rangeToLabel">
            <property name="text">
             <string>to</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="endBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Video</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Other</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bakeButton">
        <property name="text">
         <string>Bake!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="progressLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="folderButton">
        <property name="text">
         <string>Choose folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="totalProgressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="episodeProgressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMore">
    <property name="title">
     <string>More</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMore"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
